### ISSUE TITLE
你好，我使用的是ttfnet_d53_1x.py的配置文件，但是当使用darknet53.pth作为预训练模型训练时，已经训练了7个epoch，但是mAP还是很低（和不用预训练模型差不多），请问这是什么原因？

### ISSUE THREAD

**Issue by chaos0625** [ZJULearning/ttfnet#6]
Thanks for your error report and we appreciate it a lot.

**Checklist**
1. I have searched related issues but cannot get the expected help.
2. The bug has not been fixed in the latest version.

**Describe the bug**

**Reproduction**
1. What command or script did you run?
```
A placeholder for the command.
```
2. Did you make any modifications on the code or config? Did you understand what you have modified?
3. What dataset did you use?

**Environment**
 - OS: [e.g., Ubuntu 16.04.6]
 - GCC [e.g., 5.4.0]
 - PyTorch version [e.g., 1.1.0]
- How you installed PyTorch [e.g., pip, conda, source]
- GPU model [e.g., 1080Ti, V100]
- CUDA and CUDNN version
- [optional] Other information that may be related (such as `$PATH`, `$LD_LIBRARY_PATH`, `$PYTHONPATH`, etc.)

**Error traceback**
If applicable, paste the error trackback here.
```
A placeholder for trackback.
```

**Bug fix**
If you have already identified the reason, you can provide the information here. If you are willing to create a PR to fix it, please also leave a comment here and that would be much appreciated!

**Comment by liuzili97** [comment]
Hi. We have reproduced this problem, and we are working on it.

We find that the downward trend of the heatmap loss is as expected, but not for the regression loss, i.e., `losses/ttfnet_loss_wh` in the training log. This happens when using 8 GPUs for training.

You may have noticed that the `wh_head_conv_num` is `2` in ttfnet-34 and ttfnet-53 but `1` in ttfnet-18. This is because we find that larger `wh_head_conv_num` will lead to slower convergence at the beginning of training, but we can get slightly higher precision for ttfnet-34 and ttfnet-53.

So, the current solution is to set `wh_head_conv_num` to `1` in `ttfnet_d53_1x.py`, and we will check the code and update soon.

**Comment by chaos0625** [comment]
Thanks, I'll set wh_head_conv_num to 1 in ttfnet_d53_1x.py and train it again

**Comment by liuzili97** [comment]
We have adjusted the initial learning rate of the warm-up phase to solve this problem.
